.gitignore

```diff
@@ -1,2 +1,6 @@
 node_modules/
 .idea/
+
+# testing
+coverage/
+.coveralls.yml
```

Comment:
Why is `.coveralls.yml` ignored? It seems like a config file.

Reply:
mmm it sounds like I copied and pasted that... I'll remove it